prova/src/prova/Test2.java

```diff
@@ -3,9 +3,9 @@
 public class Test2 {
 	public void doTest(String x, String s) {
 		if (s.equals(x)) {
-			System.out.println("2 Eq");
+			System.out.println("2 Equals");
 		} else {
-			System.out.println("2 Not Eq");
+			System.out.println("2 Not Equals");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)